### PR TITLE
#820 Select command docs

### PR DESCRIPTION
**Pull request by vanshavenger**
1. Added notes in SELECT, FLUSHDB, DBSIZE Docs for usage related to SELECT Command.
2. use of PORT 7379 in FLUSH DB, BF.EXISTS, HRANDFIELD, JSON.ARRAPPEND, JSON.ARRINSERT, JSON.ARRPOP, JSON.ARRTRIM, JSON.DEBUG, JSON.DEL, JSON.MSET, JSON.OBJKEYS
3. bash syntax in EXISTS, HRANDFIELD,JSON.DEL,AUTH, BGREWRITEAOF, BITCOUNT,  Docs (Consistent with all DOCs)

One doubt i have 
in golang SDK the port should be 7379 instead of 6379?

cc @apoorvyadav1111 , @JyotinderSingh

### CODE REVIEW

File: docs/src/content/docs/commands/JSON.ARRINSERT.md [hunk 1]
```diff
@@ -73,11 +73,11 @@ JSON.ARRINSERT <key> <path> <index> <value> [value ...]
 Inserting at a valid index in the root path
 
 ```plaintext
```
Comment (apoorvyadav1111) [inline docs/src/content/docs/commands/JSON.ARRINSERT.md:75]:
```suggestion
```bash
```

File: docs/src/content/docs/commands/JSON.ARRINSERT.md
Comment (apoorvyadav1111) [inline docs/src/content/docs/commands/JSON.ARRINSERT.md:1]: Thanks for extending the scope of the issue. Please change all code blocks using plaintext style to bash style

File: docs/src/content/docs/commands/JSON.ARRTRIM.md
Comment (apoorvyadav1111) [inline docs/src/content/docs/commands/JSON.ARRTRIM.md:1]: same as above.

File: docs/src/content/docs/commands/JSON.DEBUG.md
Comment (apoorvyadav1111) [inline docs/src/content/docs/commands/JSON.DEBUG.md:1]: Thanks for extending the scope of the issue. Please change all code blocks using plaintext style to bash style

**Review by apoorvyadav1111 — CHANGES_REQUESTED** [review]
Hi thanks for extending the scope and auditing other documentation. I have left few comments.

**Comment by vanshavenger** [comment]
@apoorvyadav1111 throughout all the docs, We are now using bash consistently. Changed all required files.

**Comment by vanshavenger** [comment]
Any other changes required please let me know and happy to close any other DOCs issues that are stale.

**Review by apoorvyadav1111 — COMMENTED** [review]
As per standards, we use the plaintext style for Command Syntax in description. But I think let's go ahead with bash to ensure consistency and avoid confusion. Thanks for this change.

**Review by apoorvyadav1111 — APPROVED** [review]

**Pull request merged**